### PR TITLE
LPS-176421 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-176421 
https://issues.liferay.com/browse/LPS-180439
cc./ @gaborlovasdotcom 
LPS-174431 TC8 is blocked by LPS-181658
LPS-174434 TC7 is blocked by LPS-181664

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 1]
```diff
@@ -1247,6 +1247,212 @@ definition {
...
+				confirm = "true",
+				friendlyURLEdit = "/page1");
+		}
+
+		task ("Then assert that the warning pop-up appears") {
+			Alert.viewWarningSpecific(warningMessage = "Warning:The site template page was saved with a conflicting friendly URL.");
+		}
+	}
+
+	@description = "This is a use case for LPS-174431 TC-8. Verify that the site link in the URL collision cannot be accessed when the User didn't have access to the Site made from the template."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1300]: Please add the Blocked by ticket ID to the end of the description:
"Blocked by LPS-XXXXX"

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1300]: Ok and Done

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Pull request closed without merging**